init.sh

```diff
@@ -32,7 +32,34 @@ if [ ! -d "$install_directory" ]; then
     mkdir -p "$install_directory"
 fi
 
-# Copy the contents of the starter pack repository to the installation directory
+# Check if .gitignore exists in the destination directory
+# If exists, append the contents of the source .gitignore to the destination .gitignore
+if [ -f "$install_directory/.gitignore" ]; then
+    echo "ACTION REQUIRED: .gitignore already exists in the destination directory. Check the contents before saving the file!"
+    read -p "Do you want to append the list of ignored files for Sphinx docs to the existing .gitginore? Enter 'n' to skip. (y/n): " confirm
+    if [ "$confirm" = "y" ]; then
+        echo "Appending contents to the existing .gitignore..."
+        cat "$temp_directory/sp-files/.gitignore" >> "$install_directory/.gitignore"
+    else
+        echo "Operation skipped by the user. No .gitignore rule is created for Sphinx docs."
+    rm "$temp_directory/sp-files/.gitignore"
+fi
+
+# Check if Makefile exists in the destination directory
+if [ -f "$install_directory/Makefile" ]; then
+    echo "ACTION REQUIRED: Makefile already exists in the destination directory. Check the contents before running the targets!"
+    read -p "Do you want to create the Sphinx docs Makefile as a backup file? Enter 'n' to skip. (y/n): " confirm
```

Comment:
I'm not sure this is what @rkratky meant - what makes more sense imo is to make a backup of the existing Makefile and then append our Makefile to the existing Makefile.